docs/advanced-features/tabulator.md

```diff
@@ -88,10 +91,29 @@ Console, Athena views, or JDBC connectors. This is especially useful for
 customers who want to access Tabulator from external services, such as Tableau
 and Spotfire.
 
+### Continuing After Errors
+
+> Available since Quilt Platform version 1.58
+
+If Tabulator encounters an error while processing a file, it normally stops and
+returns an error. However, if `continue_on_error` is set to true, it processes
+the remaining files and records any issues in the `$issue` column.
+
+![Tabulator Issue](../imgs/tabulator-issue.png)
+
+If a file is missing a specified column:
+
+- If `nullable` (the default), the column is filled with null, while other
+  columns for all rows remain populated.
```

Comment:
@drernie this is the new universal behavior, not related to `continue_on_error`

Reply:
Sorry, can you clarify?  Are you saying that, in this release, even when continue_on_error is false, then missing columns are filled with nulls?

So the only reasons they'd need 'continue_on_error' is:

1. they have somehow set a column as non-nullable
2. they have a different error than a missing column

Is that right?

Reply:
> Sorry, can you clarify? Are you saying that, in this release, even when continue_on_error is false, then missing columns are filled with nulls?

right

> So the only reasons they'd need 'continue_on_error' is:
> 
>     1. they have somehow set a column as non-nullable
> 
>     2. they have a different error than a missing column
> 
> 
> Is that right?

yep (a common case would be invalid data in csv, e.g. non-numeric strings in numeric columns)